.github/workflows/test.yml

```diff
@@ -10,18 +10,12 @@ jobs:
       max-parallel: 5
       matrix:
         python-version: ['3.8', '3.9', '3.10', '3.11', 'pypy-3.8']
-        django-version: ['3.2', '4.0', '4.1']
+        django-version: ['3.2', '4.0', '4.1', '4.2']
         include:
           # Tox configuration for QA environment
           - python-version: '3.10'
             django-version: 'qa'
           # Django main
-          - python-version: '3.8'
-            django-version: 'main'
-            experimental: true
-          - python-version: '3.9'
-            django-version: 'main'
-            experimental: true
```

Comment:
The main branch of Django is for version 5.0 which dropped support for Python 3.8 and 3.9: https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django